packages/mint-components/src/components/sqm-close-button/sqm-close-button.feature

```diff
@@ -0,0 +1,31 @@
+@owner:coleton
+@author:coleton
+
+Feature: Popup close button
+
+  Close button component for closing popup widgets
+
+  @motivating
+  Scenario: Close button is not rendered if the widget is not a POPUP widget
+    Given the sqm-close-button component is included in widget template
+    And the widget is rendered as an EMBED widget
+    Then the close button is hidden
+
+  @motivating
+  Scenario: Clicking the close button hides the POPUP widget
+    Given the sqm-close-button component is included in widget template
+    And the widget is rendered as a POPUP widget
+    And the POPUP widget is showing
+    When the close button is clicked
+    Then the POPUP widget is hidden
+
+  @motivating
+  @ui
+  Scenario Outline: Color can be customized
+    Given a user is viewing the close button
+    And the prop "background-color" has <value>
```

Comment:
Why background colour instead of just the colour of the "X" @lillyszatmary @noahwc